circuits/benchmarks/scripts/run_benchmarks.sh

```
@@ -138,9 +150,21 @@ REPO_ROOT="$(cd "${BENCHMARKS_DIR}/../.." && pwd)"
 # Circuits live under circuits/bin (bin_dir is relative to benchmarks dir, e.g. ../bin)
 CIRCUITS_BASE_DIR="$(cd "${BENCHMARKS_DIR}/${BIN_DIR}" && pwd)"
 
-# results_<mode>_agg | results_<mode>_no_agg (see benchmark_output_dir.sh)
+# Resolve the committee for the output dir name. Explicit --committee wins; otherwise we
+# read what's currently on disk (the build step below will respect that selection). Sourced
+# here so OUTPUT_DIR can include the committee axis (`results_<mode>_<agg|no_agg>_<name>`).
+# shellcheck source=load_default_committee.sh
+source "${SCRIPT_DIR}/load_default_committee.sh"
```

Shellcheck (0.11.0)

[info] Not following: load_default_committee.sh was not specified as input (see shellcheck -x).

(SC1091)